DataHubService.js

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+
+const headers = {Accept: 'application/ld+json, application/json'};
+
+export class DataHubService {
+  constructor({
+    urls = {
+      base: '/data-hubs'
+    }
+  } = {}) {
+    this.config = {urls};
+  }
+
+  /**
+   * @method create
+   * @param {Object} options
+   * @param {string} options.baseUrl - the base baseUrl
+   * @param {string} options.config - the data hub's configuration
+   * @description returns the configuration for the newly created data hub.
```

Comment:
Could untag descriptions and put at top here and below.

Reply:
Fixed